alc/alu.cpp

```diff
@@ -866,7 +866,7 @@ void CalcPanningAndFilters(Voice *voice, const float xpos, const float ypos, con
          * scaling.
          */
         std::transform(coeffs.begin(), coeffs.end(), coeffs.begin(),
-            std::bind(std::multiplies<float>{}, _1, (1.0f-coverage)*scales[0]));
+            [coverage, scales](float a){ return a * ((1.0f-coverage)*scales[0]); });
```

Comment:
`scales` is an `const std::array&`, so that should be taken by reference.

Reply:
good point.